internal/cli/core/list.go

```diff
@@ -72,7 +72,12 @@ func GetList(inst *rpc.Instance, all bool, updatableOnly bool) []*rpc.PlatformSu
 
 	result := []*rpc.PlatformSummary{}
 	for _, platform := range platforms.GetSearchOutput() {
-		if !all && platform.InstalledVersion == "" {
+		// If both `all` and `updatableOnly` are set, `all` takes precedence.
+		if all {
```

Comment:
this could be moved outside the loop (if `all` is set you can do directly `result = platforms.GetSearchOutput()`)